src/game/mod.rs

```diff
@@ -24,6 +49,23 @@ impl GameId {
         let id: u128 = rng.gen();
         GameId(id)
     }
+    // Added for API to create a GameId object to input to the GameManager
+    pub fn from(id: &String) -> Result<Self> {
```

Comment:
Probably need a discussion about whether or not this should be consistent with `impl Display for GameId` (i.e. with the prefix or not)